features/openhab-addons/src/main/resources/footer.xml

```diff
@@ -1,4 +1,113 @@
 	<!-- aggregated features -->
+	<feature name="openhab-binding-bluetooth" description="Bluetooth Binding" version="${project.version}">
```

Comment:
Here seems to be something wrong, you're duplicating all information in this file.

Reply:
Something wrong indeed, I'll fix it.

Reply:
Strange, I've fixed it on my branch but it doesn't show it as fixed here...

Reply:
nvm, fixed it